teams_notifier.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import requests
+
+from typing import Any, Dict
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+HEADERS = {
```

Comment:
Probably it would be better to move HEADERS inside a class, because it's a part of stuff, required to send a message. Or if you use it just once, I would substitute it on a place of usage